index.html

```diff
@@ -64,6 +64,36 @@
         </gu-column>
       </gu-row>
 
+      <h2>Boxes</h2>
+      <gu-row>
+        <gu-column span="4">
+          <gu-box variant="primary">
+            This is some content
+          </gu-box>
+        </gu-column>
+        <gu-column span="4" gu-box="secondary">
```

Comment:
OK if you think this works then let's go with it, we can always keep an eye on these concerns as these components start getting used!